.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: python
 dist: focal
 python:
-  - "3.5"
```

Comment:
Can you add 3.7, 3.8 and 3.9 as well? Hopefully it just works and do not require any extra effort. This will confirm it works as is on these versions